### PR TITLE
Issue 41 feat/UI user info

### PR DESCRIPTION
**Pull request by N00BKU**

### CODE REVIEW

File: src/components/molecules/UserInfo/UserInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+import { Icon, Text } from '../..';
+
+import './UserInfo.scss';
+
+export const UserInfo = ({ date, city, hobbies }) => {
```
Comment (Veryyapeee) [inline src/components/molecules/UserInfo/UserInfo.jsx:6]: you could use just the informationRow component I guess?

File: src/components/molecules/UserInfo/UserInfo.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
+@import '../../../styles/variables.scss';
+@import '../../../styles/mixins.scss';
+
+.molecule__user__info{
```
Comment (Veryyapeee) [inline src/components/molecules/UserInfo/UserInfo.scss:4]: I gues that that's not responsive at all :D

File: src/components/molecules/UserInfo/UserInfo.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    return (
+        <div className='molecule__user__info'>
+            <Text type='primary' size='extraLarge'>User Details</Text>
+            <div>
+                <Icon iconName='birthday' size='medium' />
+                <Text type='primary' size='large'>{date}</Text>
```
Comment (Veryyapeee) [inline src/components/molecules/UserInfo/UserInfo.jsx:12]: date that we would gate from API would be in UTC ISO8 format I guess, we have to format that date

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/components/molecules/UserInfo/UserInfo.jsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
```
Comment (T0ffik) [inline src/components/molecules/UserInfo/UserInfo.jsx:1]: Looks good for me but if this is picture and text form header, component should accept onClick props too

**Review by T0ffik — APPROVED** [review]

**Review by wikbro7650 — APPROVED** [review]

File: src/components/molecules/UserInfo/UserInfo.jsx (hunk 4 above)
Comment (N00BKU) [reply src/components/molecules/UserInfo/UserInfo.jsx:1]: Can u tell me how i can implement this?

Comment (T0ffik) [reply src/components/molecules/UserInfo/UserInfo.jsx:1]: Just as children prop

**Review by Veryyapeee — APPROVED** [review]

**Pull request merged**